_summaries/Mish_ShiviAgarwal_Summary.md

```diff
@@ -0,0 +1,45 @@
+Mish: A Self Regularized Non-Monotonic Activation Function, Misra; 2019 – Summary
+
+author: Shivi Agarwal
+score:
+
+Core idea:  Efficiency of Mish Activation functions over Relu, Leaky Relu.
+
+
+This paper discusses Mish, a novel self regularized non-monotonic activation function inspired by the self gating property of Swish. Mish is mathematically defined as: f(x) = x tanh (so ft plus(x)). It is used to overcome the problem of Dying ReLU which leads to gradient information loss due to collapse in negative inputs to zero. For tasks in Computer Vision,  Mish tends to match or improve the performance of neural network architectures . 
+
+Experiment setup
+•	Task: Validate Activation functions on Image classification
```

Comment:
I think bullet point formatting might be a bit weird in the actual .md file with the bullet points.